frontend/components/DataField.vue

```diff
@@ -1,39 +1,32 @@
 <template>
   <div>
     <div class="fieldContainer">
-      <div class="companyIcon noselect">
+      <div class="companyIcon border-2 rounded-md border-zinc-600">
         <img
           class="imgIcon"
           src="../assets/images/blumilklogo.png"
           alt="Company Logo Icon"
         />
       </div>
-      <div class="companyName noselect">
+      <div class="companyName text-zinc-900">
         <span>{{ companyName }}</span>
       </div>
-      <div class="course noselect">
-        <span>{{ course }}</span>
-      </div>
-      <div class="companyInfo noselect">
+      <div class="companyInfo text-zinc-700">
         <div class="city">
           <img src="../assets/icons/locationIcon.svg" alt="locationicon" /><span
```

Comment:
Why course has the same icon as localization? And I'm not sure, if `course` is a good name. Maybe` field of study` fits better?

Reply:
Icons are placeholders for now.